exercise-src/react-native/06-custom-components/02-problem/App.tsx

```diff
@@ -0,0 +1,44 @@
+import type { FC } from "react"
+import { SafeAreaView, ScrollView, Text, View } from "react-native"
+
+const states = [
+  {
+    name: "Illinois",
+    short: "IL",
+  },
+  {
+    name: "Wisconsin",
+    short: "WI",
+  },
+]
+
+export type ListItemProps = {}
+
+export const ListItem: FC = ({}) => {}
+
+export const StateList: FC = () => {
+  return (
+    <View>
+      {states?.length > 0 ? (
+        states.map((state) => <Text key={state.short}>{state.name}</Text>)
+      ) : (
+        <Text>No states found</Text>
+      )}
+    </View>
```

Comment:
Be sure to look at the diffs and make sure they make sense. The setup diff shows this changing, and it looks like there’s a missing `<ScrollView>`.